### ISSUE TITLE
Weird light effect with SuperDuperVanilla Shaders active 

### ISSUE THREAD

**Issue by PreciseModpacker** [shiroroku/LootBeams#40]
Forge 1.19.2 43.3.0
Oculus 1.6.9
Rubidium 0.6.2c

When I'm using the SuperDuperVanilla Shaders https://www.curseforge.com/minecraft/shaders/super-duper-vanilla-shaders with the Bloom activated a weird light bloom happens:

![imagen](https://github.com/shiroroku/LootBeams/assets/110386565/ca82a961-db48-438d-b2e6-51fd4a330fd4)

![imagen](https://github.com/shiroroku/LootBeams/assets/110386565/40575ba0-5a0d-45a5-9cbc-b7ec6251c3d2)

You can disable the Bloom effect to disable that effect
![imagen](https://github.com/shiroroku/LootBeams/assets/110386565/590b8084-2e64-4f83-9074-1c16bd694a27)

Anyway it still exists a weird white circle on the ground
![imagen](https://github.com/shiroroku/LootBeams/assets/110386565/024926ae-af3d-4010-8ee3-1ba93d15d370)

I suppose it would make more sense to publish this issue on the Shader author's github, if so, tell me and I'll create another ticket there without a problem :)

**Comment by amoooooo** [comment]
yeah, make an issue on the shader's page, this isnt really something to do with loot beams specifically as other shaders work fine
